src/frontend/src/stores/events.ts

```diff
@@ -8,10 +8,17 @@ function sortByStartDate(a: Event, b: Event) {
 export const useEventStore = defineStore('events', {
   state: () => ({
     events: [] as Event[],
-    selectedEvent: null as Event | null
+    id: null as number | null
   }),
   getters: {
-    selectedEventCars: (state) => state.selectedEvent?.cars
+    selectedEvent(state) {
+      return state.events.find((event) => {
+        return event.id == state.id;
+      });
+    },
```

Comment:
One could argue it's more efficient to store events as a dict of ID to event, but it doesn't really matter